main.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
```

Comment:
We shouldn't have this file, especially not here at the root. I know you're copying this over from react, but this is distracting (main is typically the entry point to modules so seeing this at the root of the repo is confusing). Only take what we need and integrate it where we need it (we only need transform in jsx-internal, and we'll never specify sourcemap so you can simplify... hell, you could just use `require('react-tools').transform` in jsx-internal)